packages/server/graphql/mutations/createGitHubTaskIntegration.ts

```diff
@@ -68,44 +71,77 @@ export default {
       })
     }
 
-    // RESOLUTION
     const {content: rawContentStr, meetingId} = task
-    const viewerAuth = await dataLoader.get('githubAuth').load({teamId, userId: viewerId})
-
-    if (!viewerAuth) {
+    const [viewerAuth, assigneeAuth, team] = await Promise.all([
+      dataLoader.get('githubAuth').load({teamId, userId: viewerId}),
+      userId && dataLoader.get('githubAuth').load({teamId, userId}),
+      dataLoader.get('teams').load(teamId)
+    ])
+    const auth = assigneeAuth ?? viewerAuth
+    if (!auth) {
       return standardError(
-        new Error(`Assignment failed! The assignee does not have access to GitHub`),
+        new Error(`Assignment failed! Neither you nor the assignee has access to GitHub`),
         {userId: viewerId}
       )
     }
+    const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+    const viewer = teamMembers.find(({userId}) => userId === viewerId)
+    const assignee = userId && teamMembers.find((user) => user.userId === userId)
+
+    if (!viewer || (userId && !assignee)) {
```

Comment:
we already know the viewer is on the team because we checked that in L56

Reply:
task.userId is guaranteed to be an active team member because that is validated in the create/update task mutations. i think we can remove this validation check